netinet/tcp.h does not compile

I'm building libstdc++ with Watt-32 in order to enable the [C++ Networking TS](https://cplusplus.github.io/networking-ts/draft.pdf), which will likely be standardized at some point (there is some stuff in `<experimental/net>` already but I don't believe it's complete yet).

The build completes with no errors but configure fails to define macro `_GLIBCXX_HAVE_NETINET_TCP_H`.  In the logs I find the following:

```
configure:76163: checking netinet/tcp.h usability
configure:76163: /home/jw/build-gcc/script/destdir-hack.sh /home/jw/watt32-test/i386-pc-msdosdjgpp /home/jw/build-gcc/build/djcross-gcc-10.3.0/djcross/./gcc/xgcc -B/home/jw/build-gcc/build/djcross-gcc-10.3.0/djcross/./gcc/ -B/home/jw/watt32-test/i386-pc-msdosdjgpp/bin/ -B/home/jw/watt32-test/i386-pc-msdosdjgpp/lib/ -isystem /home/jw/watt32-test/i386-pc-msdosdjgpp/include -isystem /home/jw/watt32-test/i386-pc-msdosdjgpp/sys-include    -c -g -O2  conftest.c >&5
In file included from conftest.c:146:
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/netinet/tcp.h:41:9: error: unknown type name 'u_long'
   41 | typedef u_long  tcp_seq;
      |         ^~~~~~
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/netinet/tcp.h:42:9: error: unknown type name 'u_long'
   42 | typedef u_long  tcp_cc;                 /* connection count per rfc1644 */
      |         ^~~~~~
In file included from /home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/netinet/tcp.h:46,
                 from conftest.c:146:
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/sys/pack_on.h: In function 'W32_CLANG_PACK_WARN_OFF':
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/sys/pack_on.h:60:13: error: expected declaration specifiers before '#pragma'
   60 |     #pragma pack(push,1)
      |             ^~~~
In file included from conftest.c:146:
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/netinet/tcp.h:71:2: error: expected declaration specifiers before ';' token
   71 | };
      |  ^
In file included from /home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/netinet/tcp.h:73,
                 from conftest.c:146:
/home/jw/watt32-test/i386-pc-msdosdjgpp/watt/inc/sys/pack_off.h:64:13: error: expected declaration specifiers before '#pragma'
   64 |     #pragma pack(pop)
      |             ^~~~
```

I suppose `netinet/tcp.h` should include `sys/wtypes.h` and `sys/cdefs.h`?

--- a/inc/netinet/tcp.h
+++ b/inc/netinet/tcp.h
@@ -38,6 +38,9 @@
 #ifndef __NETINET_TCP_H
 #define __NETINET_TCP_H
 
+#include <sys/cdefs.h>
+#include <sys/wtypes.h>
+
 typedef u_long  tcp_seq;
 typedef u_long  tcp_cc;                 /* connection count per rfc1644 */
 

